devfile.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: jbang
+projects:
+  -
+    name: jbang
+    source:
+      type: git
+      location: "https://github.com/maxandersen/jbang.git"
+components:
+  -
+    type: chePlugin
+    id: redhat/java/latest
+  -
+    type: dockerimage
+    alias: gradle
+    image: quay.io/eclipse/che-java11-gradle:nightly
+    env:
+      - name: GRADLE_USER_HOME
+        value: /home/gradle/.gradle
+      - name: JAVA_OPTS
+        value: "-XX:MaxRAMPercentage=50 -XX:+UseParallelGC -XX:MinHeapFreeRatio=10
```

Comment:
how can I avoid having to specify this in every devfile ? I don't want to maintain this...

Reply:
I do not think it is possible atm - one need to explicitly specify dedicated env vars